lib/salus/report.rb

```diff
@@ -187,9 +136,44 @@ def export_report
 
     private
 
-    # Wrap lines to 100 chars by default
-    def wrap(text, line_width = 100)
-      text.gsub(/(.{1,#{line_width}})/, "\\1\n")
+    def deep_stringify_keys(datum)
```

Comment:
Can we just use the native ruby method? https://apidock.com/rails/Hash/deep_stringify_keys ?

Reply:
That's in ActiveSupport, not Ruby 😁I'm okay with bringing in ActiveSupport if you are, but for one method i felt this was fine

Reply:
Oh yeah good point. Bring it in, I'm sure we'll use it elsewhere anyway.